src/partials/form-404.hbs

```diff
@@ -16,6 +16,14 @@ const handleSubmit = (e) => {
 e.preventDefault();
 const formData = new FormData(e.target)
 
+const date = new Date();
+let possibleBot = false;
+if(date.value.toString().includes("GMT+0000 (Coordinated Universal Time)")){
+  possibleBot = true;
+}
+
+formData.set('possibleBot', possibleBot)
```

Comment:
Do we need to create a hidden input?

```html
<form data-netlify="true" id="notfound" name="notfoundForm" method="POST" netlify-honeypot="bot-field">
  <input class="hidden" type="hidden" name="form-name" value="notfoundForm" />
  <p class="hidden">
    <label>
      Beep-Boop. Bot-field <input name="bot-field" />
    </label>
  </p>
  <input id="url" class="hidden" name="url" />
  <input id="date" class="hidden" name="date" />
  <input type="hidden" name="possibleBot" id="possible-bot" />
  <button type="submit" id="submit-button" class="hidden"></button>
</form>
```

Then we can do the check directly on submit:

```js
window.addEventListener("DOMContentLoaded", (event) => {
  const form = document.getElementById('notfound');
  form.addEventListener('submit', handleSubmit);
  document.getElementById('url').value = window.location.href;
  document.getElementById('date').value = new Date();
  document.getElementById('possible-bot').value = new Date().toString().includes("GMT+0000 (Coordinated Universal Time)");
  document.getElementById('submit-button').click();
});
```

Reply:
Sure! that works too